CHANGELOG.md

```diff
@@ -1,36 +1,33 @@
 # CHANGELOG
 
-## Cardano SL 1.3.0 (Mainnet)
```

Comment:
The same code is running mainnet, testnet and staging. No need for the "Mainnnet" qualification here.